types/crpc/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for crpc 0.2
+// Project: https://github.com/billinghamj/crpc
+// Definitions by: Alexander Forbes-Reed <https://github.com/0xdeafcafe>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
+
+export type Client = <TReq, TRes>(path: string, body: TReq, options?: {} | null) => Promise<TRes>;
+
+export default function crpc(baseUrl: string, options?: {}): Client;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`.

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).